website/src/releases/2023-10/new-theme.mdx

```diff
@@ -0,0 +1,16 @@
+---
+date: 2023-10-06
+title: New main navigation for the Merchant Center
+description: The new style of the main navigation in Merchant Center is now available.
+type: announcement
+topics:
+  - Merchant Center
+  - Development
+---
+
+We're excited to introduce the newly redesigned main navigation for a smoother and more intuitive user experience in the Merchant Center. 
+This update brings a fresh look and enhanced functionality to help you navigate through your application effortlessly.
+
+To get started with the new navigation, simply update the `ui-kit` and `app-kit` dependencies to the latest versions.
+
+As always, if you have questions or feedback, open a GitHub [discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or [issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```

Comment:
Hi @chloe0592, good work with the release. I've added the suggestion to streamline the terms used and align with the [release notes guidelines](https://commercetools.atlassian.net/wiki/spaces/DOCS/pages/105818548/Writing+release+notes). Please check if the intent is still intact.

```suggestion
---
date: 2023-10-06
title: Introduced the new main navigation menu for the Merchant Center
description: The new main navigation menu in the Merchant Center is now available.
type: announcement
topics:
  - Merchant Center
  - Development
---

The new main navigation menu brings a fresh look and enhanced functionality to help you navigate your application effortlessly. It provides a smoother and intuitive user experience in the Merchant Center. 

To use the new navigation menu, update the `ui-kit` and `app-kit` packages to the latest versions.

As always, if you have questions or feedback, open a GitHub [discussion](https://github.com/commercetools/merchant-center-application-kit/discussions) or [issue](https://github.com/commercetools/merchant-center-application-kit/issues).
```